src/Service.GraphQLBuilder/GraphQLUtils.cs

```diff
@@ -225,56 +219,6 @@ public static ObjectType UnderlyingGraphQLEntityType(IType type)
-        /// Translates a JSON string or number value defined in the runtime configuration to a GraphQL {Type}ValueNode which represents
-        /// the associated GraphQL type. The target value type is referenced from the passed in parameterDefinition which
-        /// holds database schema metadata.
-        /// </summary>
-        /// <param name="defaultValueFromConfig">String representation of default value defined in runtime config.</param>
-        /// <param name="parameterDefinition">Database schema metadata for stored procedure parameter which include value and value type.</param>
-        /// <returns>Tuple where first item is the string representation of a GraphQLType (e.g. "Byte", "Int", "Decimal")
-        /// and the second item is the GraphQL {type}ValueNode </returns>
-        /// <exception cref="DataApiBuilderException">Raised when parameter casting fails due to unsupported type.</exception>
-        public static Tuple<string, IValueNode> ConvertValueToGraphQLType(string defaultValueFromConfig, ParameterDefinition parameterDefinition)
-        {
-            string paramValueType = SchemaConverter.GetGraphQLTypeFromSystemType(type: parameterDefinition.SystemType);
-
-            try
-            {
-                Tuple<string, IValueNode> valueNode = paramValueType switch
-                {
-                    BYTE_TYPE => new(BYTE_TYPE, new IntValueNode(byte.Parse(defaultValueFromConfig))),
-                    SHORT_TYPE => new(SHORT_TYPE, new IntValueNode(short.Parse(defaultValueFromConfig))),
-                    INT_TYPE => new(INT_TYPE, new IntValueNode(int.Parse(defaultValueFromConfig))),
-                    LONG_TYPE => new(LONG_TYPE, new IntValueNode(long.Parse(defaultValueFromConfig))),
-                    STRING_TYPE => new(STRING_TYPE, new StringValueNode(defaultValueFromConfig)),
-                    BOOLEAN_TYPE => new(BOOLEAN_TYPE, new BooleanValueNode(bool.Parse(defaultValueFromConfig))),
-                    SINGLE_TYPE => new(SINGLE_TYPE, new SingleType().ParseValue(float.Parse(defaultValueFromConfig))),
-                    FLOAT_TYPE => new(FLOAT_TYPE, new FloatValueNode(double.Parse(defaultValueFromConfig))),
-                    DECIMAL_TYPE => new(DECIMAL_TYPE, new FloatValueNode(decimal.Parse(defaultValueFromConfig))),
-                    DATETIME_TYPE => new(DATETIME_TYPE, new DateTimeType().ParseResult(
-                        DateTime.Parse(defaultValueFromConfig, DateTimeFormatInfo.InvariantInfo, DateTimeStyles.AssumeUniversal))),
-                    BYTEARRAY_TYPE => new(BYTEARRAY_TYPE, new ByteArrayType().ParseValue(Convert.FromBase64String(defaultValueFromConfig))),
-                    LOCALTIME_TYPE => new(LOCALTIME_TYPE, new LocalTimeType().ParseResult(LocalTimePattern.ExtendedIso.Parse(defaultValueFromConfig).Value)),
-                    _ => throw new NotSupportedException(message: $"The {defaultValueFromConfig} parameter's value type [{paramValueType}] is not supported.")
```

Comment:
this code is not deleted fully, just moved to GraphQLStoredProcedureBuilder class.